test-utils/state-viewer/src/main.rs

```diff
@@ -307,107 +300,118 @@ fn apply_chain_range(
-                        &mut chain_store,
-                        runtime_adapter.as_ref(),
-                        block.header().prev_hash(),
-                        shard_id,
-                    )
-                    .unwrap();
-                runtime_adapter
-                    .apply_transactions(
-                        shard_id,
-                        chunk_inner.prev_state_root(),
-                        height,
-                        block.header().raw_timestamp(),
-                        block.header().prev_hash(),
-                        block.hash(),
-                        &receipts,
-                        chunk.transactions(),
-                        chunk_inner.validator_proposals(),
-                        prev_block.header().gas_price(),
-                        chunk_inner.gas_limit(),
-                        &block.header().challenges_result(),
-                        *block.header().random_value(),
-                        true,
-                        is_first_block_with_chunk_of_version,
-                        None,
-                    )
-                    .unwrap()
-            } else {
-                let chunk_extra = chain_store
-                    .get_chunk_extra(block.header().prev_hash(), shard_id)
-                    .unwrap()
-                    .clone();
-
-                runtime_adapter
-                    .apply_transactions(
-                        shard_id,
-                        chunk_extra.state_root(),
-                        block.header().height(),
-                        block.header().raw_timestamp(),
-                        block.header().prev_hash(),
-                        &block.hash(),
-                        &[],
-                        &[],
-                        chunk_extra.validator_proposals(),
-                        block.header().gas_price(),
-                        chunk_extra.gas_limit(),
-                        &block.header().challenges_result(),
-                        *block.header().random_value(),
-                        false,
-                        false,
-                        None,
-                    )
-                    .unwrap()
-            };
-            let (outcome_root, _) =
-                ApplyTransactionResult::compute_outcomes_proof(&apply_result.outcomes);
-            let chunk_extra = ChunkExtra::new(
-                &apply_result.new_root,
-                outcome_root,
-                apply_result.validator_proposals,
-                apply_result.total_gas_burnt,
-                near_config.genesis.config.gas_limit,
-                apply_result.total_balance_burnt,
-            );
+        let maybe_block_hash = chain_store.get_block_hash_by_height(height);
+        if maybe_block_hash.is_err() {
+            info!(target:"state-viewer", "Skipping block #{}, because it's not available in ChainStore.", height);
+            continue;
+        }
+        let block_hash = maybe_block_hash.unwrap();
```

Comment:
It is common to avoid unnecessary `unwrap`s with `if let` expressions:

```suggestion
        let block_hash = if let Ok(block_hash) = chain_store.get_block_hash_by_height(height) {
            block_hash
        } else {
            info!(target:"state-viewer", "Skipping block #{}, because it's not available in ChainStore.", height);
            continue;
        };
```

Reply:
Done, thanks.